### PR TITLE
Cleanup System.Drawing.Drawing2D code

### PR DESCRIPTION
**Pull request by hughbe**
As discussed in #20711

### CODE REVIEW

File: src/System.Drawing.Common/src/System/Drawing/Drawing2D/AdjustableArrowCap.cs [hunk 1]
```diff
@@ -2,170 +2,104 @@
...
-                throw SafeNativeMethods.Gdip.StatusException(status);
-        }
+        public AdjustableArrowCap(float width, float height) : this(width, height, true) { }
 
-        private float _GetHeight()
+        public AdjustableArrowCap(float width, float height, bool isFilled)
         {
-            float height;
-            int status = SafeNativeMethods.Gdip.GdipGetAdjustableArrowCapHeight(new HandleRef(this, nativeCap), out height);
+            int status = SafeNativeMethods.Gdip.GdipCreateAdjustableArrowCap(height, width, isFilled, out IntPtr nativeCap);
```
Comment (stephentoub) [inline src/System.Drawing.Common/src/System/Drawing/Drawing2D/AdjustableArrowCap.cs]: Another case where I'd prefer if we not declare as part of the out.  Takes me a while looking around the source to find where nativeCap is declared. (I'll stop commenting on these.)

File: src/System.Drawing.Common/src/System/Drawing/Drawing2D/AdjustableArrowCap.cs [hunk 2]
```diff
@@ -2,170 +2,104 @@
...
+                    throw SafeNativeMethods.Gdip.StatusException(status);
+
+                return height;
+            }
+            set
+            {
+                int status = SafeNativeMethods.Gdip.GdipSetAdjustableArrowCapHeight(new HandleRef(this, nativeCap), value);
+
+                if (status != SafeNativeMethods.Gdip.Ok)
+                    throw SafeNativeMethods.Gdip.StatusException(status);
```
Comment (stephentoub) [inline src/System.Drawing.Common/src/System/Drawing/Drawing2D/AdjustableArrowCap.cs]: Didn't you add a CheckStatus method or something that could be used here?

File: src/System.Drawing.Common/src/System/Drawing/Drawing2D/AdjustableArrowCap.cs (hunk 1 above)
Comment (hughbe) [inline src/System.Drawing.Common/src/System/Drawing/Drawing2D/AdjustableArrowCap.cs]: Oh yeah sorry I should have reverted these changes - they were sent in before your comments elsewhere

File: src/System.Drawing.Common/src/System/Drawing/Drawing2D/AdjustableArrowCap.cs (hunk 2 above)
Comment (hughbe) [inline src/System.Drawing.Common/src/System/Drawing/Drawing2D/AdjustableArrowCap.cs]: That stuff is in #21180 unfortunately

**Comment by hughbe** [comment]
Updated to revert the changes to use out variables

**Comment by hughbe** [comment]
Test OSX x64 Debug Build
Test Windows x64 Debug Build

**Pull request merged**